Update backup documentation for foremanctl (#4850)
theforeman/foreman-documentation: guides/common/modules/ref_available-backup-methods.adoc

= Available backup methods

[role="_abstract"]
ifdef::containerized[]
You can create an offline backup of your {ProjectServer}.
endif::[]
ifndef::containerized[]
Choose an offline backup method based on how long you can keep {Project} services unavailable and how much time you can allocate for the backup.
endif::[]

Offline backup::
All {Project} services are stopped during an offline backup to ensure data consistency.
ifdef::containerized[]
The backup process includes the following actions:
* Preflight checks to verify that no tasks are running and that the database is consistent.
* Service shutdown to stop all {Project} services cleanly.

> * Database dumps to create PostgreSQL dumps of all databases.

Reviewer: This sounds a bit off. Maybe make it shorter?

```suggestion
* PostgreSQL database dumps to backup content and configuration.
```

* Configuration backup to archive the {foremanctl} state and configuration.
* Content backup to back up Pulp content directory.
This is an optional action. You can skip this action by using the `--skip-pulp-content` option with the `{foremanctl} backup` command.

> * Service restart to restore all services to running state.

Reviewer: ```suggestion
* Service restart to restore all {Project} services.
```

Your call.
